molgenis-api/src/main/java/org/molgenis/api/VersionController.java

```diff
@@ -0,0 +1,43 @@
+package org.molgenis.api;
+
+import static java.time.ZonedDateTime.now;
+import static java.util.Objects.requireNonNull;
+import static org.molgenis.api.ApiNamespace.API_PATH;
+import static org.springframework.web.bind.annotation.RequestMethod.OPTIONS;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.Instant;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(API_PATH)
+public class VersionController extends ApiController {
```

Comment:
How about renaming this class to RootApiController so we can extend it with other planned endpoints such as listing all apis with their versions?